qutebrowser/browser/adblock.py

```diff
@@ -61,15 +60,18 @@ def get_fileobj(byte_io):
     return byte_io
 
 
-def is_whitelisted_host(host):
-    """Check if the given host is on the adblock whitelist.
+def is_whitelisted_host(url):
```

Comment:
This should probably be renamed to `is_whitelisted_url`, but I'll take care of it.